how-to/postgrest.md

```diff
@@ -397,4 +397,189 @@ Now, you can perform a keyword-based search directly from your web browser. Simp
 ]
 ```
 
-This seamless and user-friendly approach to searching through your data with keywords is both practical and exciting. It's a fantastic way to enhance your search capabilities, don't you think?
\ No newline at end of file
+This seamless and user-friendly approach to searching through your data with keywords is both practical and exciting. It's a fantastic way to enhance your search capabilities, don't you think?
+
+
+## Using Keyword Auto Complete
+
+PGroonga has features to implement auto complete which is explained in [this how to section](https://pgroonga.github.io/how-to/auto-complete.html).
```

Comment:
Could you use relative link to reuse the same link path for all languages? And could you use named link (`[label][link-id]` and `[link-id]: URL`) instead of inline link (`[label][URL]`) for easy to translate?

```suggestion
PGroonga has features to implement auto complete which is explained in [the auto complete how to section][auto-complete].
```

and put the following to the page footer:

```markdown
[auto-complete]: auto-complete.html
```